azure-pipelines.yml

```diff
@@ -301,17 +297,18 @@ jobs:
               echo "$(git diff $(branch_sha) $(fork_sha) -- LATEST | tail -n+6 | grep '^\+' | cut -c2-)"
           }
 
-          if is_trigger_commit; then
-              setvar is_release true
-              setvar trigger_sha $(branch_sha)
-              setvar release_sha "$(added_line | awk '{print $1}')"
-              setvar release_tag "$(added_line | awk '{print $2}')"
+          if changes_release_files; then
+              if changes_one_line_in_latest; then
```

Comment:
doesn’t this break this PR which adds more than one line?

Reply:
No, because it also changes more than the 2 release files, so it doesn't get there.

Reply:
ah right, thanks for the explanation!